docs/topics/js/js-project-setup.md

```diff
@@ -159,21 +159,17 @@ these transitive dependencies as well.
 The dependencies on the [standard library](https://kotlinlang.org/api/latest/jvm/stdlib/index.html)
 are added automatically. The version of the standard library is the same as the version of the Kotlin Multiplatform plugin.
 
-The [`kotlin.test`](https://kotlinlang.org/api/latest/kotlin.test/) API is available for multiplatform tests.
-When you create a multiplatform project, the Project Wizard automatically adds test dependencies to all the source sets.
-
-If you don't use the Project Wizard to create your project, you can add the dependencies manually:
+For multiplatform tests, the [`kotlin.test`](https://kotlinlang.org/api/latest/kotlin.test/) API is available.
+When you create a multiplatform project, you can add test dependencies to all the source sets:
```

Comment:
The description seems to contradict the code: “add dependencies” although we only add one dependency. The ambiguity is resolved by the comment, but I'd say better not to have it at all and reinforce the message with the comment.

I'd expand with something like “...you can add test dependencies to all the source sets by adding a single dependency in `commonTest`.”